app/controllers/dashboards_controller.rb

```diff
@@ -1,4 +1,30 @@
 # frozen_string_literal: true
 class DashboardsController < ApplicationController
-  def index; end
+  include Flashable
+
+  def index
+    @uploads = Upload.last_uploads
+  end
+
+  def build
+    @version = InstitutionBuilder.run(current_user)
```

Comment:
@mphprogrammer what is returned if the build fails? I don't see any error handling.